server/graphql/v2/object/Host.js

```diff
@@ -1137,6 +1145,130 @@ export const GraphQLHost = new GraphQLObjectType({
           };
         },
       },
+      hostedAccounts: {
+        type: new GraphQLNonNull(GraphQLAccountCollection),
+        description: 'Returns a list of accounts hosted by this host',
+        args: {
+          ...getCollectionArgs({ limit: 100, offset: 0 }),
+          accountType: { type: new GraphQLList(GraphQLAccountType) },
+          isApproved: {
+            type: GraphQLBoolean,
+            description: 'Filter on (un)approved collectives',
+            defaultValue: true,
+          },
+          isFrozen: {
+            type: GraphQLBoolean,
+            description: 'Filter on frozen accounts',
+            defaultValue: false,
+          },
+          isUnhosted: {
+            type: GraphQLBoolean,
+            description: 'Filter on unhosted accounts',
+            defaultValue: false,
+          },
+          hostFeesStructure: {
+            type: GraphQLHostFeeStructure,
+            description: 'Filters on the Host fees structure applied to this account',
+          },
+          searchTerm: {
+            type: GraphQLString,
+            description:
+              'A term to search membership. Searches in collective tags, name, slug, members description and role.',
+          },
+          orderBy: {
+            type: GraphQLOrderByInput,
+            description: 'Order of the results',
+          },
+        },
+        async resolve(host, args, req) {
+          const isAdmin = req.remoteUser?.isAdmin(host.id);
+          if (!isAdmin) {
+            throw new Unauthorized('You need to be logged in as an admin of the host to see its hosted accounts');
+          }
+
+          const where = {
+            HostCollectiveId: host.id,
+          };
+
+          if (args.accountType && args.accountType.length > 0) {
+            where.type = {
+              [Op.in]: args.accountType.map(value => AccountTypeToModelMapping[value]),
+            };
+          }
+
+          if (!isNil(args.isFrozen)) {
+            if (args.isFrozen) {
+              set(where, `data.features.${FEATURE.ALL}`, false);
+            } else {
+              set(where, `data.features.${FEATURE.ALL}`, { [Op.is]: null });
+            }
+          }
```

Comment:
Would it be sufficient to only add this condition when isFrozen is `true`? Since isFrozen has a default value of false, it will always add this condition, so would be nice to avoid if possible

```suggestion
          if (args.isFrozen) {
            set(where, `data.features.${FEATURE.ALL}`, false);
          }
```

Reply:
Not really, because this would prevent us from removing frozen collectives from the active tab.